client.go

```diff
@@ -33,10 +34,16 @@ const (
 )
 
 var (
-	newline       = []byte{'\n'}
-	notFoundMsg   = []byte(`{"e":"not_found"}`)
-	disconnectMsg = []byte(`{"data": {"":{"@system":{"client_disconnect":true}}}}`)
-	clearStateMsg = []byte(`{"c":1}`)
+	newline          = []byte{'\n'}
+	notFoundMsg      = []byte(`{"e":"not_found"}`)
+	disconnectMsg    = []byte(`{"data": {"":{"@system":{"client_disconnect":true}}}}`)
+	clearStateMsg    = []byte(`{"c":1}`)
+	STATE_CREATED    = "CREATED"
```

Comment:
The idiomatic way to represent a state (enumeration) would be a `const` value (not `var`) generated by `iota`(are those values in variables needed?).

Reply:
This was easier to debug, but since it works, can be refactored into `iota`.